libs/designsystem/src/lib/components/page/page.component.spec.ts

```diff
@@ -171,9 +162,9 @@ describe('PageComponent', () => {
       expect(leaveEmitSpy).toHaveBeenCalledTimes(1);
     });
   });
+
   async function triggerOnLeave(router: SpyObject<Router>) {
     spectator.detectChanges();
```

Comment:
Please add comment as to why `spectator.detectChanges` is needed - or put it in a beforeEach for all nav tests...